### PR TITLE
v3 address-validation release

### PR DESCRIPTION
**Pull request by dsteeve**
Add release note for new release of address-validation to production

### CODE REVIEW

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Released version 3 of address validation. Updates include:
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:1]:
```suggestion
We released version 3 (v3) of address validation. Updates include:
```

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+- `/validate` and `/candidate` endpoints
+
+     - Removed support for US Congressional District.
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:4]: Should this be a sub-bullet  of 'validate' and 'candidate above? Or its own bullet point?
```suggestion
     - Removing support for US Congressional District. 
```

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Released version 3 of address validation. Updates include:
+- `/validate` and `/candidate` endpoints
+
+     - Removed support for US Congressional District. 
+     - Added country code support for both International Standards Organization (ISO)  and Federal Information Processing (FIPS) standards.
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:5]: Should this be a sub-bullet of 'validate' and 'candidate above? Or its own bullet point?

```suggestion
     - Adding country code support for both International Standards Organization (ISO) and Federal Information Processing (FIPS) standards.
```

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+- `/cityStateProvince` endpoint is renamed to `/locality` to reflect the capability.
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:7]: Reflect the capability of what?

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+- For ease of mapping,  fields are renamed to be identical between this service and contact information service version 2.
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:9]:
```suggestion
- For ease of mapping,  fields are renamed to be identical between this service and contact information service in version 2 of the API.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]

File: content/verification/address-validation/release-notes/2024-05-15.md (hunk 4 above)
Comment (dsteeve) [reply content/verification/address-validation/release-notes/2024-05-15.md:7]: reflects the capability of the endpoint.

File: content/verification/address-validation/release-notes/2024-05-15.md (hunk 3 above)
Comment (dsteeve) [reply content/verification/address-validation/release-notes/2024-05-15.md:5]: yes, these are sub-bullets describing changes to the 2 endpoints.

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+We released version 3 (v3) of address validation. Updates include:
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:1]:
```suggestion
We released version 3 (v3) of the Address Validation API. Updates include:
```

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+- `/validate` and `/candidate` endpoints
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:2]:
```suggestion
- Adding `/validate` and `/candidate` endpoints.
```

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+     - Removing support for US Congressional District.
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:4]: @dsteeve are these two (Removing support...) and (Adding country code..) supposed to be sub-bullets of '/validate' and '/candidate'? Just want to double check.

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
+We released version 3 (v3) of address validation. Updates include:
+- `/validate` and `/candidate` endpoints
+
+     - Removing support for US Congressional District. 
+     - Adding country code support for both International Standards Organization (ISO) and Federal Information Processing (FIPS) standards.
+
+- `/cityStateProvince` endpoint is renamed to `/locality` to reflect the capability of the endpoint.
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:7]:
```suggestion
- Renaming `/cityStateProvince` endpoint to `/locality` to reflect the capability of the endpoint.
```

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+
+- For ease of mapping,  fields are renamed to be identical between this API and Contact Information API version 2 (v2).
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:9]:
```suggestion
- For ease of mapping, fields are renamed to be identical between this API and Contact Information API version 2 (v2).
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
Requested a few more changes.

File: content/verification/address-validation/release-notes/2024-05-15.md (hunk 7 above)
Comment (dsteeve) [reply content/verification/address-validation/release-notes/2024-05-15.md:2]: these endpoints existed in v1 of their API, so I don't think we can say they are being added here.  v3 basically takes functionality of v1 and v2 and combines them into v3 and describes the major changes such as congressional district and countrycode ISO style that is now different in v3 vs v1

File: content/verification/address-validation/release-notes/2024-05-15.md (hunk 8 above)
Comment (dsteeve) [reply content/verification/address-validation/release-notes/2024-05-15.md:4]: They are sub-bullets to these endpoints.  If you look at address-validation v1, there was a congressional district param, and that is no longer there, and country code handling is now different as well.

Comment (joeabraun) [reply content/verification/address-validation/release-notes/2024-05-15.md:4]: Awesome, I'll resolve!

File: content/verification/address-validation/release-notes/2024-05-15.md (hunk 7 above)
Comment (joeabraun) [reply content/verification/address-validation/release-notes/2024-05-15.md:2]: Got it, I can remove this comment.

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 11]
```diff
@@ -0,0 +1,11 @@
+We released version 3 (v3) of the Address Validation API. Updates include:
+- `/validate` and `/candidate` endpoints
+
+     - Removing support for US Congressional District. 
+     - Adding country code support for both International Standards Organization (ISO) and Federal Information Processing (FIPS) standards.
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:5]:
```suggestion
     - Adding country code support for both International Standards Organization (ISO) and Federal Information Processing Standards (FIPS). 
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
One last change then should be good to go

File: content/verification/address-validation/release-notes/2024-05-15.md (hunk 11 above)
Comment (joeabraun) [reply content/verification/address-validation/release-notes/2024-05-15.md:5]: @dsteeve I believe this is the Federal Information Processing Standards, so it should all be capitalized?

File: content/verification/address-validation/release-notes/2024-05-15.md [hunk 12]
```diff
@@ -0,0 +1,11 @@
+We released version 3 (v3) of the Address Validation API. Updates include:
+- `/validate` and `/candidate` endpoints
```
Comment (joeabraun) [inline content/verification/address-validation/release-notes/2024-05-15.md:2]:
```suggestion
- `/validate` and `/candidate` endpoints.
```

**Review by joeabraun — COMMENTED** [review]
Adding a period

**Review by joeabraun — APPROVED** [review]
Approved! Looks good to me!

**Pull request merged**